pkg/report/utils.go

```diff
@@ -36,22 +36,22 @@ func InitDibReport(dir string) (*Report, error) {
 		BuildReports:   []BuildReport{},
 	}
 
-	// Create Report root directory
-	if err := os.MkdirAll(path.Join(dibReport.Dir, dibReport.Name), 0o755); err != nil {
-		return nil, err
-	}
+	return &dibReport
+}
 
-	// Create Report build logs directory
-	if err := os.MkdirAll(dibReport.GetBuildLogsDir(), 0o755); err != nil {
-		return nil, err
-	}
+// CreateRootDir Create Report root directory.
+func (r Report) CreateRootDir() error {
```

Comment:
It seems that this function is not used anywhere

As we use `os.MkdirAll` function, we do not need to create rootdir anymore

Reply:
Removed functions to create dir and use directly o.MkdirAll